content/publication/maranghi-2022-ai/index.md

```diff
@@ -40,7 +40,10 @@ image:
 projects: []
 publishDate: '2023-02-06T11:14:04.668002Z'
 publication_types:
-- '2'
-abstract: ''
+- '3'
+abstract: 'The Associazione Medici Diabetologi (AMD) col- lects and manages one of the largest worldwide- available collections of diabetic patient records, also known as the AMD database. This paper presents the initial results of an ongoing project whose focus is the application of Artificial Intel- ligence and Machine Learning techniques for con- ceptualizing, cleaning, and analyzing such an im- portant and valuable dataset, with the goal of pro- viding predictive insights to better support diabetol- ogists in their diagnostic and therapeutic choices.'
```

Comment:
inherited dashes and newlines from the article

Reply:
done